packages/core/content-manager/admin/src/utils/validation.ts

```diff
@@ -317,7 +317,7 @@ const addMinValidation: ValidationFn =
         (attribute.type === 'component' && attribute.repeatable) ||
         attribute.type === 'dynamiczone'
       ) {
-        if (options.status !== 'draft' && !attribute.required && 'test' in schema && min) {
+        if (options.status === 'draft' && !attribute.required && 'test' in schema && min) {
```

Comment:
I'm a bit confused by this, are we saying minimum validations of repeatable components and dzs happen on drafts too? 

If so should we remove the status check all together so it runs on both draft and publish?

Reply:
There is the empty array condition that we need only for drafts. I have another issue to fix still though so I'll change this again

- null/undefined => ok -> Value when creating
- empty array => ok -> Value after creation returned by the api